src/core/load-fragment.c

```diff
@@ -5421,6 +5422,61 @@ int config_parse_ip_filter_bpf_progs(
         return 0;
 }
 
+int config_parse_bpffs_programs(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+        _cleanup_free_ char *resolved = NULL;
+        _cleanup_free_ char *bpffs_path = NULL;
+        enum bpf_attach_type attach_type;
+
+        const Unit *u = userdata;
+        CGroupContext *c = data;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(c);
+
+        if (isempty(rvalue)) {
+                while (c->bpffs_programs)
+                        cgroup_context_free_bpffs_program(c, c->bpffs_programs);
+                return 0;
+        }
+
+        r = unit_full_printf(u, rvalue, &resolved);
+        if (r < 0) {
+                log_syntax(unit, LOG_WARNING, filename, line, r, "Failed to resolve unit specifiers in '%s', ignoring: %m", rvalue);
+                return 0;
+        }
+
+        r = bpffs_program_from_string(resolved, &attach_type, &bpffs_path);
+        if (r < 0) {
+                log_syntax(unit, LOG_WARNING, filename, line, r, "Failed to resolve pinned BPF program for '%s': %m", resolved);
```

Comment:
add `, ignoring", since you are ignoring the thing here

Reply:
addressed